frontend/app/src/screens/proposed-changes/proposed-change-details.tsx

```diff
@@ -23,23 +24,28 @@ import { useAtom } from "jotai";
 import { HTMLAttributes } from "react";
 import { useNavigate, useParams } from "react-router-dom";
 import { TaskDisplay } from "../branches/task-display";
+import LoadingScreen from "../loading-screen/loading-screen";
 import { getObjectPermissionsQuery } from "../permission/queries/getObjectPermissions";
 import { getPermission } from "../permission/utils";
-import {
-  BRANCH_MERGE_WORKFLOW,
-  BRANCH_REBASE_WORKFLOW,
-  BRANCH_VALIDATE_WORKFLOW,
-} from "../tasks/constants";
+import { PROPOSED_CHANGE_MERGE_WORKFLOW } from "../tasks/constants";
 
 export const ProposedChangeDetails = ({ className, ...props }: HTMLAttributes<HTMLDivElement>) => {
   const { proposedChangeId } = useParams();
   const [proposedChangesDetails] = useAtom(proposedChangedState);
 
   const navigate = useNavigate();
 
-  const { loading, data, error } = useQuery(
-    gql(getObjectPermissionsQuery(PROPOSED_CHANGES_OBJECT))
-  );
+  const { data } = useQuery(gql(getObjectPermissionsQuery(PROPOSED_CHANGES_OBJECT)), {
+    pollInterval: 2000,
+  });
```

Comment:
I'm not sure to understand why pollInterval here?

Reply:
to get the latest updates and change the UI accordingly

since we disable the merge button depending on the "merge" tasks, we could also update the proposed changes informations to mark it as merged and make it consistent if someone else merges a proposed change